corehq/apps/data_cleaning/tasks.py

```
@@ -116,13 +116,27 @@ def _prune_records_and_complete_session(session, errored_doc_ids):
     session.save(update_fields=['completed_on'])
 
 
-def _create_case_blocks(session, records):
+def _create_case_blocks(session, records, errored_doc_ids):
     blocks = []
     case_ids = [rec.doc_id for rec in records]
     cases = {c.case_id: c for c in CommCareCase.objects.get_cases(case_ids, session.domain)}
     for record in records:
         case = cases[record.doc_id]
-        update = record.get_edited_case_properties(case)
+        try:
+            update = record.get_edited_case_properties(case)
+        except Exception as e:
```

can we add a comment justifying the catch-all exception here?